python/ray/serve/storage/kv_store.py

```diff
@@ -0,0 +1,251 @@
+import sqlite3
+
+import boto3
+import ray.experimental.internal_kv as ray_kv
+from botocore.exceptions import ClientError
+from ray.serve.storage.kv_store_base import KVStoreBase
+from ray.serve.utils import logger
+
+
+def get_storage_key(self, key):
+    """In case we need to access kvstore
+    """
+    return "{ns}-{key}".format(ns=self.namespace, key=key)
```

Comment:
Is there any simple way to keep this in sync with the implementation of this function in the different Serve KV store classes?  If we change this format in the future, we'll have to remember to update it in multiple places, but maybe it's not a big deal.

Reply:
i think we should keep single source of truth on this as well, and based on your PR it seems we have the need to get storage key without an kv store instance. In this case i think we should just keep this function as the only one for imports like this, and for other kv store types, its interface requires implementation of get_storage_key so we can expect it to be consistent as long as it's using the same class.